earth_enterprise/rpms/build.gradle

```diff
@@ -463,7 +472,6 @@ task openGeeCommonDeb (type: GeeDeb, dependsOn: openGeeCommonInitScripts) {
     requires('libstdc++6')
     requires('libtool')
     requires('libxml2-utils')
-    requires('openssl')
```

Comment:
this removal is done because PostGIS does not need SSL ? I can see the case where a client connecting to Postgres needs SSL on client and server side. Unless this is a dup since Open SSL is already deployed by GEE server.

Reply:
This is just some cleanup. Notice that this is the section of code that builds the debs, not the rpms. The rpm section doesn't have this requirement, so I think it was left here by mistake.